content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +60,140 @@ olm-operator       1/1     1            1           5m52s
-metadata:
-  name: og-single
-  namespace: default
-spec:
-  targetNamespaces:
-  - default
+## User Interface (Running the Console Locally)
+
+Use the admin console (compatible with upstream Kubernetes) to interact with and visualize the resources managed by OLM. Create subscriptions, approve install plans, identify Operator-managed resources, and more.
+
+Ensure `kubectl` is pointing at a cluster and run:
 
-$ kubectl apply -f operatorgroup.yaml
-operatorgroup.operators.coreos.com/og-single created
+```shell
+make run-console-local
 ```
 
-Then create a subscription for the etcd operator:
+Then visit `http://localhost:9000` to view the console.
+
+## Customizing OLM installations 
+
+Deployments of OLM can be stamped out with different configurations by writing a `values.yaml` file and running commands to generate resources.
+
+Here's an example `values.yaml`:
+
+```yaml
+# sets the apiversion to use for rbac-resources. Change to `authorization.openshift.io` for openshift
+rbacApiVersion: rbac.authorization.k8s.io
+# namespace is the namespace the operators will _run_
+namespace: olm
+# watchedNamespaces is a comma-separated list of namespaces the operators will _watch_ for OLM resources.
+# Omit to enable OLM in all namespaces
+watchedNamespaces: olm
+# catalog_namespace is the namespace where the catalog operator will look for global catalogs.
+# entries in global catalogs can be resolved in any watched namespace
+catalog_namespace: olm
+# operator_namespace is the namespace where the operator runs
+operator_namespace: operators
+# OLM operator run configuration
+olm:
+  # OLM operator doesn't do any leader election (yet), set to 1
+  replicaCount: 1
+  # The image to run. If not building a local image, use sha256 image references
+  image:
+    ref: quay.io/operator-framework/olm:local
+    pullPolicy: IfNotPresent
+  service:
+    # port for readiness/liveness probes
+    internalPort: 8080
+# catalog operator run configuration
+catalog:
+  # Catalog operator doesn't do any leader election (yet), set to 1
+  replicaCount: 1
+  # The image to run. If not building a local image, use sha256 image references
+  image:
+    ref: quay.io/operator-framework/olm:local
+    pullPolicy: IfNotPresent
+  service:
+    # port for readiness/liveness probes
+    internalPort: 8080
+```
 
-```sh
-$ cat subscription.yaml
+To configure a release of OLM for installation in a cluster:
+
+1. Create a `my-values.yaml` like the example above with the desired configuration or choose an existing one from this repository. The latest production values can be found in [deploy/upstream/values.yaml][deploy-upstream-values].
+
+1. Generate deployment files from the templates and the `my-values.yaml` using `package_release.sh`
+
+   ```bash
+   # first arg must be a semver-compatible version string
+   # second arg is the output directory
+   # third arg is the values.yaml file
+   ./scripts/package_release.sh 1.0.0-myolm ./my-olm-deployment my-values.yaml
```

Comment:
This script does not exist in this repo, we should probably either copy paste the script in a separate file in the doc site so that users can copy it to create their own file(this is my preference so that users don't have to jump back and forth between the site and the repos), or link to the script at least to begin with.

Reply:
I am not sure if I am following that. I understand that:
- I would be reading the info in https://olm.operatorframework.io/ (which is the doc ref)
- However, locally I would have the OLM repo cloned since the goal here is to install the OLM project. Then, I would have this script because it is provided there. I mean, I do not think that has a reason for someone that is following the getting started docs to have the docs locally instead of the OLM project. 

Then, if we move the script to OLM docs and if changed in the repo it will be providing the wrong info. Is that make sense?